src/Microsoft.AspNetCore.Razor.Language/Intermediate/FieldDeclarationIRNode.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Razor.Language.Intermediate
+{
+    public sealed class FieldDeclarationIRNode : MemberDeclarationIRNode
+    {
+        private ItemCollection _annotations;
+
+        public override ItemCollection Annotations
+        {
+            get
+            {
+                if (_annotations == null)
+                {
+                    _annotations = new DefaultItemCollection();
+                }
+
+                return _annotations;
+            }
+        }
+
+        public override IList<RazorIRNode> Children { get; } = new List<RazorIRNode>();
```

Comment:
Yeah that's going to be a more complex topic. I'm going to make these not have children for now.